app/controllers/comment_controller.php

```diff
@@ -0,0 +1,55 @@
+<?php
+class CommentController extends AppController
+{
+	CONST MAX_ITEM_PER_PAGE = 10;
+
+	public function view()
+	{
+		session_start();
```

Comment:
you can call session_start() in bootstrap so you won't need to call it on every controller action